pages/api/chat.ts

```diff
@@ -48,7 +48,9 @@ const handler = async (req: NextRequest): Promise<Response> => {
       temperatureToUse = DEFAULT_TEMPERATURE;
     }
 
-    const isValidModel = Object.values(OpenAIModelID).toString().includes(model.id)
+    model.id = "gpt-4"
```

Comment:
Was this meant to be hardcoded? I felt like this was debug code, but I might just not be following.

Reply:
no lol good catch - was for testing purposes. Just removed.